Meta/import-wpt-test.py

```diff
@@ -115,14 +115,13 @@ def map_to_path(sources: list[ResourceAndType], is_resource=True, resource_path=
         base_directory = test_type.input_path if source.type == ResourceType.INPUT else test_type.expected_path
 
         if source.resource.startswith('/') or not is_resource:
-            file_path = base_directory + '/' + source.resource
+            file_path = base_directory.rstrip('/') + '/' + source.resource.lstrip('/')
```

Comment:
Another solution would be to use `Path`, it handles this type of thing quite nicely.

Reply:
I've changed this function to use `Path` objects rather than manual string concatenations. I've kept `lstrip` in the one place it's required.